account_bank_statement_import_auto_reconcile/models/account_bank_statement_import.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import _, api, fields, models
+
+
+# pylint: disable=R7980
+class AccountBankStatementImport(models.TransientModel):
+    _inherit = 'account.bank.statement.import'
+
+    auto_reconcile = fields.Boolean('Auto reconcile', default=True)
```

Comment:
Customer thinks this would be better off by default, and I agree. It's not a disaster if someone leaves it off by default and realizes he would be better off having it on, because he can just delete the bank statement and try again. But if it's on and he would have wanted it to be off, recovery is much more tedious.

Aside from that, this is a bit of a lone checkbox. Should we make it configurable on the Bank account journal instead, like we did now for the [batch payments](https://github.com/OCA/bank-statement-import/pull/175/files#diff-9c7d7b289511979198dd286f0ee48534R11)? Another configuration option "Split statements on import" is also there, so it seems like a logical place.

Reply:
if there are no automatic reconciliation rules configured for the journal, this should do nothing, right?

this checkbox is meant to be able to temporarily turn off the automatic reconciliation, if you never want it, just don't configure rules on the journal